src/lib/components/chips/Chips.component.js

```diff
@@ -0,0 +1,114 @@
+//@flow
+import * as defaultTheme from "../../style/theme";
+import styled, { css } from "styled-components";
+import React from "react";
+import { lighten } from "polished";
+import { mergeTheme } from "../../utils";
+
+type Props = {
+  text: string,
+  color?: String,
+  variant?: Variant,
+  icon?: Node,
+  buttonIcon?: Node,
+  onClick?: any => void,
+};
+
+const defaultProps = {
+  variant: "base",
+  icon: null,
+  buttonIcon:null,
+  text: "",
+};
+
+const ChipsContainer = styled.div`
+  display: inline-flex;
+  background-color:${defaultTheme.grayLighter};
+  border-radius: 15px;
+  cursor: pointer;
+
+  ${props => {
+    const brandingTheme = mergeTheme(props.theme, defaultTheme);
+
+    const brandLight = lighten(0.1, brandingTheme[props.variant]).toString();
+    console.log(brandingTheme);
+    
+    return css`
+      background-color: ${brandingTheme[props.variant]};
+      color: ${props.color};
+    
+      &:hover {
```

Comment:
hover and active should be needed only when onClick is defined. 
« pointer: cursor » should be added to the hover as well

Reply:
Can I add those when onRemove is defined as well?

Reply:
I think the Remove button should have its own hover style so that in the case when we have onRemove and onClick, we know what is clicked when clicking.